packages/appkit/tests/client/adapter-management.test.ts

```
@@ -7,13 +8,22 @@ import { AppKit } from '../../src/client.js'
 import { mockBitcoinAdapter } from '../mocks/Adapter.js'
 import { bitcoin } from '../mocks/Networks.js'
 import { mockOptions } from '../mocks/Options.js'
+import { mockUniversalProvider } from '../mocks/Providers.js'
 import { mockBlockchainApiController, mockStorageUtil, mockWindowAndDocument } from '../test-utils'
 
 mockWindowAndDocument()
 mockStorageUtil()
 mockBlockchainApiController()
 
 describe('Adapter Management', () => {
+  beforeEach(() => {
+    vi.spyOn(UniversalProvider, 'init').mockResolvedValue(mockUniversalProvider)
+  })
+
+  afterEach(() => {
+    vi.clearAllMocks()
+  })
```

I'm adding this since i saw a lot of silent errors when running tests

---

What are those silent errors? Would better to share the exact reason we need them pls 🙏🏽 @magiziz